<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>848</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>guibabel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox_avail_serialports"/>
       </item>
       <item>
        <widget class="QPushButton" name="button_refresh_serialports">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="button_connect_disconnect_serialport">
       <property name="text">
        <string>connect serialport</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Packets valid:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_valid">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>---</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>invalid:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_invalid">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>---</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_logger">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>485</width>
      <height>170</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>485</width>
      <height>170</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_fft">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>815</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_plotter">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>815</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_connect_disconnect_serialport">
   <property name="text">
    <string>connect serialport</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
